build_automation/content_management/api_views.py

```diff
@@ -253,14 +256,15 @@ def create(self, request, *args, **kwargs):
             }
             return Response(data, status=status.HTTP_409_CONFLICT)
 
+
 class MetadataMatchViewSet(ViewSet):
-    queryset = Content.objects.values_list('name', flat=True);
+    queryset = Content.objects.values_list('name', flat=True)
     '''print(list(queryset))'''
-    def getNames(self, request):    
-        fileNameArray = list(queryset)
-        data =  {
-            content_files: fileNameArray
+
+    def get_names(self, request):
```

Comment:
flake8 was complaining that this should lowercase. This should fit more with the python's style